scripts/start.sh

```diff
@@ -148,9 +148,8 @@ retry $MAX_WAIT host_check_ksqlDBserver_up || exit 1
 echo -e "\nRun ksqlDB queries (takes about 1 minute):"
 ${DIR}/ksqlDB/run_ksqlDB.sh
 
-echo -e "\nStart consumers for additional topics: WIKIPEDIANOBOT, EN_WIKIPEDIA_GT_1_COUNTS"
+echo -e "\nStart additional consumer for topic WIKIPEDIANOBOT"
```

Comment:
This script previously started up a consumer for `EN_WIKIPEDIA_GT_1_COUNTS`, which was the _final_ stream in the pipeline. This appears to now be equivalent to `EN_WIKIPEDIA_GT_1`. Do we want to keep the consumer for the final output here?

Reply:
Added back in